waku/waku_store/client.nim

```diff
@@ -59,3 +59,22 @@ proc query*(
     return err(StoreError(kind: ErrorCode.PEER_DIAL_FAILURE, address: $peer))
 
   return await self.sendStoreRequest(request, connection)
+
+proc queryToAny*(
+    self: WakuStoreClient, request: StoreQueryRequest, peerId = none(PeerId)
+): Future[StoreQueryResult] {.async.} =
+  ## This proc is similar to the query one but in this case
+  ## we don't specify a particular peer and instead we get it from peer manager
```

Comment:
> Love it! Idk why we didn't have something like this before. Maybe it would make sense to in the future use this for the REST API if a store node is not specified?

Yes something to consider in the future :)